src/main/java/TaskTwo.java

```diff
@@ -23,4 +25,16 @@ public static String getExpandString(String[] array) {
     public static String[] splitString(String inputString) {
         return inputString.split("");
     }
+
+    public static String inputString() {
```

Comment:
As long as inputString() method returns a value it's worth to name it with a "get" prefix: getUserInput();